canonical-web-and-design/jaas.ai
# Switch out sass-lint for stylelint ✨

## Done

Switch out sass-lint for stylelint ✨

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Sanity check code
- Ensure all tests pass

Fixes #428
